logging/logging.go

```diff
@@ -66,6 +67,13 @@ func PrintCurLog() {
 // VeryVerbose = optional verbosity
 // Error = stderr
 func SetupLogs() {
+	// initialize logging flags
+	if glog.V(2) {
```

Comment:
If I understand correctly https://godoc.org/github.com/golang/glog#V, the function returns true if the verbosity setting is at least 2. So, any value >=2 will do. I will check though.